### DISCUSSION TITLE
Ideas on APKEditor

### DISCUSSION THREAD

**Discussion by REAndroid** [REAndroid/APKEditor#1] [Ideas]
Please share your ideas / thoughts on this tool 

To start here is mine:
- Include refactor option to refactor resource names
- Load resource names from **public.xml**

**Comment by timea-techgirl** [comment]
![image](https://user-images.githubusercontent.com/7058205/209222271-bb4b97f1-433e-4938-a96f-07476a15e437.png)

Tried v1.0.7. Renamed resources don't have standard names.

**Reply by REAndroid** [reply]
Yes, This is the next goal. I am planing to implement smart refactoring from values like:

``` html
<string name="abcd">No internet connection</string>
```
will be renamed as 
``` html
<string name="no_internet_connection">No internet connection</string>
````

and for colors
``` html
<color name="abcd">#FF0000</color>
```
will be renamed as 
``` html
<color name="red">#FF0000</color>
````

But your resources seem don't need refactoring

**Reply by timea-techgirl** [reply]
That's great. :)
The picture on the left is the result from another tool.

Colors looks like this from its result,
![image](https://user-images.githubusercontent.com/7058205/209248163-6a1d31bd-7485-41af-a355-955d4a89d071.png)

**Reply by REAndroid** [reply]
Looks like your app refactored names from **R$--** (R.java) dex classes , this project do not have means to access dex classes

**Reply by timea-techgirl** [reply]
It's just apktool that renames it this way

> Looks like your app refactored names from **R$--** (R.java) dex classes , this project do not have means to access dex classes
